framework/include/base/MooseBase.h

```diff
@@ -11,6 +11,15 @@
 
 #include <string>
 
+#define usingMooseBaseMembers                                                                      \
```

Comment:
happy to see MooseBase getting some love
let me know if you think of a better name for this class

Reply:
seems fine to me